styles.css

```diff
@@ -547,3 +549,181 @@ a {
   margin-left: 130px;
   margin-top: -50px;
 }
+
+/* Contact Form section */
+
+.form_section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 120px;
+  padding-left: 24px;
+  padding-right: 24px;
+  gap: 58px;
+  background: #fff;
+  margin-top: 1650px;
+}
+
+.form_text {
+  display: flex;
+  justify-content: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  margin: 0 0 30px 0;
+  width: 327px;
+}
+
+.contact_me_form {
+  margin: 120px 0 0 20px;
+  text-align: center;
+  width: 327px;
+  padding-top: 20px;
+  opacity: 2;
+}
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), please let's use Flexbox to layout the form elements. 👍